frontend-js/style.css

```diff
@@ -93,3 +93,31 @@ a {
 .app-auth-btn {
   margin: 0 auto;
 }
+
+.categories-page {
+  display: grid;
+  justify-content: center;
+  align-content: center;
+}
+.categories-title {
+  text-align: center;
```

Comment:
text-align is not needed since container should already take care of centering this.

Reply:
But he doesn't.